R/utils_connect_database.R

```diff
@@ -41,11 +41,17 @@ create_connection <- function(db, collection, url) {
 "COLLECTIONS"
 
 COLLECTIONS <- 
-  c("Stage_Profiles",
+  c("CarbonUsage",
+    "Checkinout",
+    "New_tier",
+    "PolarBear",
```

Comment:
These are all specific to the gamified version that is used in CanBikeCo and should not be in the standard dashboard. Focusing on the supplemental tables,

```
>>> COLLECTIONS=["Stage_timeseries", "Stage_analysis_timeseries", "Stage_profile"]
>>> COLLECTIONS.extend([t["name"] for t in scfg["supplementary_tables"]])
>>> COLLECTIONS
['Stage_timeseries', 'Stage_analysis_timeseries', 'Stage_profile', 'Checkinout', 'PolarBear', 'SuggestionSys', 'Tier_Sys']
```